packages/ldc/ldc-tools-utils-bugs.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 336bbdbc..4b9e8c88 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -736,12 +737,12 @@ endif()
+ #
+ # Auxiliary build and test utils.
+ #
+-add_subdirectory(utils)
++#add_subdirectory(utils)
+ 
+ #
+ # Auxiliary tools.
+ #
+-add_subdirectory(tools)
++#add_subdirectory(tools)
```

Comment:
Similar issue here, for some reason, despite specifying that I only want the `ldc2` and `ldmd2` targets built, the termux scripts would try to build this too and error out because it's not passed the right linker.

What's weird is that if I go into the termux build directory for ldc and invoke `make ldc2`, it works and ignores these tools, but if I run `build-package.sh`, it pulls these tools in somehow.  As above, since I don't want these tools, just remove them.